README.md

```diff
@@ -0,0 +1,63 @@
+# Hello-Microverse
+
+> In this project, I will copy a project from "Microverse collaboration Trials". No complex coding is required for this exercise. 
```

Comment:
- [ ] It would be better if our README corresponds to our project. The README title and description can be improved

Reply:
I have updated the title and added more descriptions. Thank you.